jwcrypto/jwe.py

```diff
@@ -289,6 +290,56 @@ def unwrap(self, key, keylen, ek, headers):
         return cek
 
 
+class _Pbes2HsAesKw(_RawKeyMgmt):
+
+    def __init__(self, hashsize, keysize):
+        self.backend = default_backend()
+        self.hashsize = hashsize
+        self.keysize = keysize // 8
+
+    def _get_key(self, alg, key, p2s, p2c):
```

Comment:
I don't like this. The input to PBES2 isn't an oct JWK it is a password string. It seems to me that conflating these two makes it easy to use a low entropy key where a salted, derived key should be used.